src/lib/components/Experts.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ExpertBlock } from './ExpertBlock';
+import cards from '../../modules/experts/mockDataCards';
+
+export const Experts: React.FC = () => {
+  const allExperts = cards.map((card) => {
+    return <ExpertBlock expert={card} key={card.phone} />;
+  });
+  return <div>{allExperts}</div>;
+};
```

Comment:
this file was moved from lib to modules/experts.
pls check it with Tania.
we should not be adding a copy of the component

Reply:
fixed